types/hbs/hbs-tests.ts

```diff
@@ -0,0 +1,26 @@
+import * as express from 'express';
+
+const app = express();
+
+app.set('view engine', 'html');
+// tslint:disable-next-line no-var-requires
+app.engine('html', require('hbs').__express);
```

Comment:
Actually, this lint rule is important -- `require()` is a function returning `any`. Please use an `import hbs = require("hbs")` instead.